gap/attr.gi

```diff
@@ -1336,6 +1336,42 @@ end);
 # It is the backend to IsBipartiteDigraph, Bicomponents, and DigraphColouring
 # for a 2-colouring
 
+InstallMethod(DigraphMycielskian, "for a symmetric digraph",
+[IsDigraph],
+function(D)
+  local n, i, j;
+  if not IsSymmetricDigraph(D) or IsMultiDigraph(D) then
```

Comment:
What does/should this do to digraphs that have loops?

Reply:
That's a good point! If we take a graph with loops like `D := Digraph([[1, 2], [1]])`, then DigraphMycielskian(D) has a repeated edge, which I don't want. I'll fix this.